src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,151 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private void validateIndex(int index) {
```

Comment:
According to [Common Mistakes,](https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md) your private methods should be placed after the public ones.